sampleApp/sample.tsx

```diff
@@ -36,28 +46,39 @@ oneNoteDataProvider.getNotebooks().then((notebooks) => {
 			notebookListUpdater: updater,
 			callbacks: {
 				onNotebookHierarchyUpdated: (newNotebookHierarchy) => {
+					let latestNotebook = newNotebookHierarchy[newNotebookHierarchy.length - 1].name;
+					if(defaultDropdownLabel === "")
```

Comment:
onNotebookHierarchyUpdated is called when the notebook tree is updated, which could be anything from a notebook to a page. Also, the added notebook is not guaranteed to be the very last element in the list. When notebook hierarchy is updated, in this case, we shouldn't change the label.